docs/topics/equality.md

```diff
@@ -34,6 +41,24 @@ Referential equality is checked by the `===` operation and its negated counterpa
 true if and only if `a` and `b` point to the same object. For values represented by primitive types at runtime
 (for example, `Int`), the `===` equality check is equivalent to the `==` check.
 
+> In Kotlin/JS, there is no difference between `===` and `==` operators when comparing strings. The reason is that in Kotlin/JS,
+> the `==` operator checks that two values are equal. Meanwhile, the `===` operator also checks that two values are equal 
+> besides verifying that the types of these two values are equal. In JS, Kotlin always uses the `===` operator:
```

Comment:
This looks like some random fact about K/JS platform implementation aspects and I believe here it would only complicate the matter. We can find a place for that in Js interop (https://kotlinlang.org/docs/js-interop.html) and maybe explain even more weird cases there like https://pl.kotl.in/N-30hX1AM, and leave only a link here.

Reply:
I placed the content in Js interop meanwhile, I'll ask in the TW team for suggestions of placement also. Meanwhile, could you please help me explaining why `println(1.0 as Any === 1 as Any)` is TRUE in Kotlin/JS and not FALSE. I was looking for explanations about this behavior but didn't find. Also, if you think we can add more edge cases, please let me know. Thanks.

Reply:
In, K/JS https://kotlinlang.org/docs/js-to-kotlin-interop.html#kotlin-types-in-javascript five numeric types are all represented with the same JS Number type in runtime, so the values of these types numerically equal are totally indistinguishable. Not sure if we can commit that it is as designed, but this is the current implementation peculiarity.